src/utils/initial-transform.ts

```
@@ -0,0 +1,81 @@
+ * Calculate the initial transform of an element relative to the viewport.
+ *
+ * @param element The element to calculate the transform for.
+ * @param offsetDistance The distance from the viewport center to the transform origin.
+ * @param maxRotation The maximum rotation of the element in degrees.
+ * @param maxZTranslation The maximum z-axis translation of the element in pixels.
+ *
+ * @returns The initial transform of the element as an object with x, y, z, rotateX, and rotateY properties.
+ */
+export const calculateInitialTransform = (
+  element: HTMLElement,
+  offsetDistance: number = 250,
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)